zrml/prediction-markets/src/lib.rs

```diff
@@ -1678,6 +1661,8 @@ mod pallet {
         BadOnInitialize,
         /// A complete set of assets has been bought \[market_id, amount_per_asset, buyer\]
         BoughtCompleteSet(MarketIdOf<T>, BalanceOf<T>, <T as frame_system::Config>::AccountId),
+        /// A failed market dispute mechansim finally resolved \[market_id\]
+        FailedDisputeMechanismResolved(MarketIdOf<T>),
```

Comment:
No extrinsic seems to use it.

Reply:
cfc576ae4f3af61f09271b3f98c1fd7f12262e1d Good catch. Depended on commits before all of Malte's reviews.